src/functions-and-arrays.js

```diff
@@ -1,19 +1,47 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+
+  
+  if (num1 > num2) {
+    return num1
+  } else {
+      return num2
+    }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+
+  let word = 0;
+  let finder = 0;
+  if (arr.length === 0) {
+    return null
+  } else { 
+  for (let i = 0; i < arr.length; i++) {
+    if (word.length < arr[i].length) {
```

Comment:
`word` is currently a number, so you can't call the length method on it (only on strings or arrays). Instead word should be a string and then you just reassign it a new value each time the condition is true:
`word= arr[i]`